### PR TITLE
Chore: Upgrading dependencies

### PR DESCRIPTION
**Pull request by mckn**
- Bump sass-loader from 13.2.0 to 13.2.1 (#126)
- Bump eslint-config-prettier from 8.5.0 to 8.8.0 (#124)
- Bump @grafana/ui from 9.1.2 to 9.5.1 (#136)
- Bump @grafana/data from 9.1.2 to 9.5.1 (#137)
- Bump @swc/jest from 0.2.20 to 0.2.23 (#139)

### CODE REVIEW

**Comment by mckn** [comment]
https://github.com/grafana/clock-panel/pull/140

File: package.json [hunk 1]
```diff
@@ -52,33 +52,37 @@
...
+    "@types/react": "17.0.42",
+    "@types/react-dom": "17.0.14",
+    "jest-environment-jsdom": "^29.3.1"
   },
   "dependencies": {
     "@emotion/css": "^11.1.3",
-    "@grafana/data": "9.1.2",
-    "@grafana/runtime": "9.1.2",
-    "@grafana/ui": "9.1.2",
+    "@grafana/data": "9.5.1",
```
Comment (jackw) [inline package.json:79]: Do we need to bump these and if so should we update the grafanaDependency in plugin.json?

**Review by jackw — APPROVED** [review]
LGTM! 🚀

File: package.json (hunk 1 above)
Comment (jackw) [reply package.json:79]: Okie dokie, let's do like this. 🚀

**Pull request merged**